packages/compiler-sfc/__tests__/compileStyle.spec.ts

```diff
@@ -270,3 +270,31 @@ describe('SFC CSS modules', () => {
     expect(result.modules!.bazQux).toBeUndefined()
   })
 })
+
+describe('SFC style preprocessors', () => {
+  test('scss @import', () => {
+    const res = compileStyle({
+      source: `
+        @import "import.scss";
+      `,
+      filename: 'test.scss',
+      id: '',
+      preprocessLang: 'scss',
+      preprocessOptions: {
+        // test with file will get error which `import.scss is not find`
+        importer: function(url: string, prev: string, done: string) {
+          if (url === 'import.scss') {
```

Comment:
I alway got error which `import.scss is not find`. I don‘t reslove the error 😭 and need you take a look.